docs/Configuration.md

```diff
@@ -145,11 +145,107 @@ oc create secret generic snow-secret \
 -n pelorus
 ```
 
+### Labels
+
+Labels are key/value pairs that are attached to objects, such as pods. Labels are intended to be used to specify identifying attributes of objects that are meaningful and relevant to users and Pelorus.
+
+The commit time, deploy time, and failure exporters all rely on labels to indentify the application that is associated with an object.  The object can
+include a build, build configuration, deployment or issue.
+
+In Pelorus the default label is: `app.kubernetes.io/name=<app_name>` where 
+app_name is the name of the application(s) being monitored. The label can
+be customized by setting the `APP_LABEL` variable to your custom value. 
+
+An example may be to override the APP_LABEL for the failure exporter to indicate a production bug or issue.
+The `APP_LABEL` with the value `production_issue/name` may give more context than `app.kubernetes.io/name`
+In this case the Github issue would be labeled with `production_issue/name=todolist`
+
+Example Failure exporter config:
+```
+- app_name: failure-exporter
+  exporter_type: failure
+  env_from_secrets:
+  - github-secret
+  extraEnv:
+  - name: LOG_LEVEL
+    value: DEBUG
+  - name: PROVIDER
+    value: github
+  - name: PROJECTS
+    value: konveyor/mig-demo-apps,konveyor/oadp-operator
+  - name: APP_LABEL
+    value: production_issue/name
+```
+
+> **Warning** 
+> If the application label is not properly configured, Pelorus will not collect data for that object.  
+>
+> One can detect a misconfigured application label by ensuring the `DEBUG` log level is set in the
```

Comment:
I don't think there are any exporters that will explicitly detect a misconfigured app label, right?

There may be some that give additional details to help troubleshoot, but will any explicitly say you have a missing app label? I know the ones which check OpenShift first _filter_ by app label, meaning it will never even know that it's missing.

Maybe we should save this for once we have a nice troubleshooting doc, as we've wished for.

Reply:
ok.. updating to just be...
> **Warning** 
> If the application label is not properly configured, Pelorus will not collect data for that object.

Reply:
👍

Reply:
@KevinMGranger please review and merge once you think your comments are addressed. I already gave here lgtm, but as you have opened request change you should close it pls.